src/components/summary-list/template.njk

```diff
@@ -6,20 +6,27 @@
     {% endif -%}
   </a>
 {% endmacro -%}
+
+{# Determine if we need 2 or 3 columns #}
+{% set hasActions = false %}
```

Comment:
I wonder if this could be clearer, perhaps something like

```suggestion
{% set anyRowHasActions = false %}
```

Reply:
Yep I'm happy with this